extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -97,6 +115,17 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	async pinNotebook(bookTreeItem: BookTreeItem, unpin: boolean = false): Promise<void> {
+		let bookPathToUpdate = bookTreeItem.book?.contentPath;
+		if (bookPathToUpdate) {
+			let pinStatusChanged = unpin ? this.bookPinManager.unpinNotebook(bookTreeItem) : this.bookPinManager.pinNotebook(bookTreeItem);
+			if (pinStatusChanged) {
+				bookTreeItem.contextValue = unpin ? 'savedNotebook' : 'pinnedNotebook';
+				vscode.window.showInformationMessage(unpin ? loc.msgBookUnpinned(bookTreeItem.title) : loc.msgBookPinned(bookTreeItem.title));
```

Comment:
I think the message is unnecessary here - that's spammy and it seems pretty clear to me already what's happening.